test/Microsoft.NET.TestFramework/Commands/PublishCommand.cs

```diff
@@ -29,7 +29,7 @@ public override CommandResult Execute(params string[] args)
             return command.Execute();
         }
 
-        public DirectoryInfo GetOutputDirectory(string targetFramework = "netcoreapp1.0", string configuration = "Debug", string runtimeIdentifier = "")
+        public override DirectoryInfo GetOutputDirectory(string targetFramework = "netcoreapp1.0", string configuration = "Debug", string runtimeIdentifier = "")
         {
             string output = Path.Combine(ProjectRootPath, "bin", configuration, targetFramework, runtimeIdentifier, PublishSubfolderName);
```

Comment:
This may be going too far, so if you don't feel like fixing it now, that's fine with me.

What do you think about this calling "base.GetOutputDirectory()" and then appending `PublishSubfolderName` to that?